src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,17 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({
+  todo: { title, completed, user },
+}) => (
+  <article className={cn('TodoInfo', {
+    'TodoInfo--completed': completed,
+  })}
+  >
+    <h2 className="TodoInfo__title">
+      {title}
+    </h2>
+
+    <UserInfo user={user} />
```

Comment:
user may not exist cause `find` method can return undefined 
we need to handle that here